### PR TITLE
[Layout] - DZ60RGB V1 - Add personal configuration

### PR DESCRIPTION
**Pull request by Didel**
## Description

This adds my personal configuration so other people can use it as well. A proper documentation of all the changes is in the works for a later moment.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dztech/dz60rgb/keymaps/didel/keymap.c [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {[0] = LAYOUT(KC_GESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS, CTL_T(KC_CAPS), KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT, LSFT_T(KC_GRAVE), KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, RSFT_T(KC_SLSH), KC_UP, LT(2, KC_DEL), KC_LCTL, LM(1, MOD_LALT), KC_LGUI, KC_SPC, KC_RALT, MO(1), KC_LEFT, KC_DOWN, KC_RGHT),
```
Comment (drashna) [inline keyboards/dztech/dz60rgb/keymaps/didel/keymap.c:3]: You may want to reformat this so ... it's readable.

File: keyboards/dztech/dz60rgb/keymaps/didel/config.h [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#pragma once
```
Comment (drashna) [inline keyboards/dztech/dz60rgb/keymaps/didel/config.h:1]: Could you add a license header here?

File: keyboards/dztech/dz60rgb/keymaps/didel/keymap.c [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/dztech/dz60rgb/keymaps/didel/keymap.c:1]: Could you add a license header here?

File: keyboards/dztech/dz60rgb/keymaps/didel/keymap.c (hunk 1 above)
Comment (Didel) [reply keyboards/dztech/dz60rgb/keymaps/didel/keymap.c:3]: Sure! I was already seeing some other layouts had fancy named layers, that it something I'll look into. Apart from that, is there an example of a layout that is considered 'readable'?

Comment (noroadsleft) [reply keyboards/dztech/dz60rgb/keymaps/didel/keymap.c:3]: The [default keymap for the board](https://github.com/qmk/qmk_firmware/blob/349646bfe0658b1b0236fcc986db714b510114df/keyboards/dztech/dz60rgb/keymaps/default/keymap.c) is a decent example.

**Comment by Didel** [comment]
@noroadsleft I tried to fix things, but I guess I only made them worse... I now fixed my master-branch to be equal to QMK's (which automatically closed this PR) and put all the changes in their own feature-branch, as I should have done to start with. I've created a new PR, see #9989 . It contains all the requested changes from this PR.

**Pull request closed without merging**